src/AccessibilityInsights.Core/Enums/FileFilters.cs

```diff
@@ -7,21 +7,7 @@ namespace Axe.Windows.Core.Enums
     /// </summary>
     public static class FileFilters
     {
-        public const string A11yFileFilter = "AccessibilityInsights files (*.a11ytest,*.a11yevent)|*.a11ytest;*.a11yevent|A11y Test files (*.a11ytest)|*.a11ytest|A11y Events files (*.a11yevent)|*.a11yevent|All files (*.*)|*.*";
-        public const string TestFileFilter = "AccessibilityInsights Test files (*.a11ytest)|*.a11ytest|All files (*.*)|*.*";
-        public const string EventsFileFilter = "AccessibiltyInsights Events files (*.a11yevent)|*.a11yevent|All files (*.*)|*.*";
         public const string TestExtension = ".a11ytest";
-        public const string EventsExtension = ".a11yevent";
         public const string SarifExtension = ".sarif";
```

Comment:
>SarifExtension [](start = 28, length = 14)

This is used in 1 place in LocationHelper and 1 place in LocationHelperUnitTests. It would be easy to replace with an internal const string on the LocationHelper class